generics/constants/common.js

```diff
@@ -93,5 +93,6 @@
   DELETE_USER: "delete-user",
   USER: "User",
   USER_DELETE_MODULE: "userDelete",
-  OFF: "OFF"
+  OFF: "OFF",
+  UPLOAD_FILE:"v1/cloud-services/files/upload"
```

Comment:
please add this for UPLOAD_FILE api/cloud-services/mlcore/v1/files/upload @praveenKDass

Reply:
Changed as api/cloud-services/mlcore/v1/files/upload